game/script-topics.rpy

```diff
@@ -1537,6 +1537,80 @@ label talk_driving:
             else:
                 n "That's what friends are for, [player]!"
 
+# Natsuki laments her inability to drive and questions the player on if they can
+init 5 python:
+    registerTopic(
+        Topic(
+            persistent._topic_database,
+            label="talk_gaming",
+            unlocked=True,
+            prompt="Are you into video games?",
+            conditional=None,
+            category=["Natsuki", "Media"],
+            player_says=True,
+            affinity_range=(jn_affinity.NORMAL, jn_affinity.LOVE),
+            location="classroom"
+        ),
+        topic_group=TOPIC_TYPE_NORMAL
+    )
+
+label talk_gaming:
+    n "Gaming?" 
+    n "Well...{w=0.3} duh!"
+    n "You bet I'm into gaming,{w=0.1} [player]!"
+    n "I wouldn't say I'm the most active player...{w=0.2} but I definitely do my share of button mashing."
+    n "Hmm..."
+    n "I don't think I even need to ask,{w=0.1} but..."
+    menu:
+        n "What about you,{w=0.1} [player]?{w=0.2} Do you play often?"
+
+        "Absolutely!":
+            $ gaming_choice = 0
```

Comment:
Ideally it might make sense to add a persistent variable for this, we can probably use this for other dialogue branching here and there.